src/session.rs

```diff
@@ -47,13 +49,15 @@ impl SessionOptions {
 impl SessionWrapper {
     #[napi]
     pub async fn create_session(options: &SessionOptions) -> napi::Result<Self> {
-        let s = SessionBuilder::new()
-            .known_node(options.connect_points[0].clone())
-            .custom_identity(get_self_identity(options))
-            .build()
-            .await
-            .map_err(err_to_napi)?;
-        Ok(SessionWrapper { inner: s })
+        let builder = configure_session_builder(options);
+        let session = builder.build().await.map_err(err_to_napi)?;
+        if let Some(keyspace) = &options.keyspace {
+            session
+                .use_keyspace(keyspace, false)
+                .await
+                .map_err(err_to_napi)?;
+        }
+        Ok(SessionWrapper { inner: session })
```

Comment:
:wrench: There is a `use_keyspace()` method on `SessionBuilder` that should be used instead of `use`ing the keyspace on an already set up `Session`.